src/schema/tags.ts

```diff
@@ -57,6 +59,11 @@ export const typeDefs = /* GraphQL */ `
   }
 
   extend type Query {
+    """
+    Get all tags
+    """
+    tags: [Keyword]
```

Comment:
Little bit awkward, but with the upcoming ones I want to keep them all in this file 😅